calibration/experiments/sphere_held_suarez_rhoe_equilmoist/postprocessing.jl

```diff
@@ -0,0 +1,106 @@
+import EnsembleKalmanProcesses as EKP
+using EnsembleKalmanProcesses.ParameterDistributions
+using EnsembleKalmanProcesses.TOMLInterface
+using Distributions
+import Statistics: mean
+import CairoMakie: Makie
+import ClimaCalibrate
+
+function convergence_plot(
+    eki,
+    prior;
+    theta_star = 65.0,
+    output = joinpath("output", "sphere_held_suarez_rhoe_equilmoist"),
+)
+    u_vec = EKP.get_u(eki)
+
+    error_vec = Float64[]
+    spread_vec = Float64[]
+    for ensemble in u_vec
+        ensemble_error = 0.0
+        ensemble_spread = 0.0
+        ensemble_mean = mean(ensemble)
+        for i in ensemble
+            ensemble_error += abs(i - theta_star)^2
+            ensemble_spread += abs(i - ensemble_mean)^2
+        end
+        ensemble_error /= length(ensemble)
+        ensemble_spread /= length(ensemble)
+
+        push!(error_vec, ensemble_error)
+        push!(spread_vec, ensemble_spread)
+    end
+
+    phi_vec = transform_unconstrained_to_constrained(prior, u_vec)
+    u_series = [getindex.(u_vec, i) for i in 1:10]
+    phi_series = [getindex.(phi_vec, i) for i in 1:10]
```

Comment:
```suggestion
    u_series = map(i->getindex.(u_vec, i), 1:10)
    phi_series = map(i->getindex.(phi_vec, i), 1:10)
```
Do we need the `.` in `getindex.(u_vec, i)`?

Reply:
The broadcast is needed, but this snippet is confusing. It is "transposing" the vector of vectors `u_vec`.

Something like this is clearer: `eachcol(reduce(vcat, u_vec))`